lib/resources/r4/medication_request.yaml

```diff
@@ -48,26 +79,88 @@ fields:
   required: 'No'
   type: code
   description: Medication Request Priority
+  action: get
   binding:
     description: Medication Request Priority
     terminology:
     - display: Request Priority
       system: http://hl7.org/fhir/request-priority
       info_link: https://hl7.org/fhir/r4/valueset-request-priority.html
+- name: doNotPerform
+  required: 'No'
+  cardinality: 0..1
+  type: boolean
+  description: Indicates whether or not the provider is asking for the medication request not to occur.
+  action: create
+  note: If provided, must be set to `false`.
+  example: |
+    {
+      "doNotPerform": false
+    }
+- name: reported[x]
+  required: 'Yes'
+  cardinality: 0..1
+  type: boolean
+  description: Indicates if this record was captured as a secondary 'reported' record rather than as an original primary source-of-truth record.
+  action: create
+  note: Must be of type boolean and must be set to `true`.
```

Comment:
Add a similar statement to read/search: The server may change the stored value following business rules and pending changes to regulation. Needs better language.

Reply:
Sorry, where exactly should this be added in read/search? Just like in the implementation notes section? Also not really sure how it should be worded. @kathypickering?

Reply:
You're adding a statement in create similar to what we added to read/search about the intent and changes pending regulatory. I believe we currently will accept either value, but it will be read back out as "order" - but in the future we will change that to "plan" following US Core guidelines

Reply:
Got it- wasn't sure since the comment was on the reported field. Is keeping it the same as the reach/search version alright? Possibly specifying that we accept order and plan for creates, like:

**Implementation Notes**

We accept "Order" and "Plan" in the intent field, but are currently always returning a value of "Order." We recommend using the reported field to determine if a medication is an authorization or a medication reported by a patient. In the future, we will support "Plan" and "Order" in accordance with the US Core Profile pending changes.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/353/commits/387ed3fddb74d45d41d8b7765a48b4f7b87ce43e#diff-152ba56eff926bdcdfebbc13bbde483fR57